### PR TITLE
Orders manager prototype

### PR DESCRIPTION
**Pull request by Sugar-pack**

### CODE REVIEW

File: internal/grpcapi/order_service.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+)
+
+type OrderService struct {
+	pb2.OrdersManagerServiceServer
+	dbConn *sqlx.DB
+}
+
+func (s OrderService) ReceiveOrder(cxt context.Context, order *pb2.Order) (*pb2.OrderTnxResponse, error) {
```
Comment (pzabolotniy) [inline internal/grpcapi/order_service.go:17]: Receive means "to get", but here we INSERT order. Suggest to rename method.

File: internal/grpcapi/order_service.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	logger := logging.FromContext(cxt)
+	logger.Info("ReceiveOrder")
+	orderID := uuid.New()
+	txID := uuid.New()
+	query := "INSERT INTO orders_db ( id,  user_id, label, created_at ) VALUES ($1, $2, $3, $4)"
```
Comment (pzabolotniy) [inline internal/grpcapi/order_service.go:22]: orders_db - suffix **_db** is a bad naming for a table.

File: sql-migrations/init_orders_db.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
+-- +migrate Up
+-- +migrate StatementBegin
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS orders_db (
+    id uuid PRIMARY KEY,
+    user_id uuid NOT NULL,
+    label varchar NOT NULL,
+    created_at varchar NOT NULL
```
Comment (pzabolotniy) [inline sql-migrations/init_orders_db.sql:9]: created_at should be time-typed, not varchar.

File: internal/grpcapi/order_service.go (hunk 1 above)
Comment (Sugar-pack) [reply internal/grpcapi/order_service.go]: Done

File: sql-migrations/init_orders_db.sql (hunk 3 above)
Comment (Sugar-pack) [reply sql-migrations/init_orders_db.sql]: Done. Proto file fixed

**Pull request merged**